src/amo/components/HomeHeroBanner/index.js

```diff
@@ -108,7 +108,7 @@ export class HomeHeroBannerBase extends React.Component<Props> {
 
           <p>
             {i18n.gettext(`Keep different parts of your online
-              life—work, personal, etc.—separated by color-coded tabs.`)}
+              life—work, personal, etc.—separated by color-coded tabs`)}
```

Comment:
@devaneymoz Pretty sure these will show up as new translators for translators; I'm not sure how fuzzy we get for strings that just have a period added/removed but this seems like a lot of work for translators for now a big change, especially as these are arguably sentences 😄 

Is it that important to remove them?

Reply:
@tofumatt the periods should go away. They were never intended to be part of these headlines. While we're not using headline title case, we are treating these as headlines sans periods. It would be great to get the style consistent.